src/components/charts/shared/shapes/Triangle.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Triangle extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.padding = 0.5;
+  }
+
+  render() {
+    const {
+      height: initHeight,
+      width: initWidth,
+      classes,
+      fillColor,
+    } = this.props;
+    const height = initHeight - this.padding;
+    const width = initWidth - this.padding;
+
+    return (
+      <polygon
+        className={classes}
+        points={`${width / 2},${this.padding} ${width},${height} 0,${height}`}
+        fill={fillColor}
+        stroke={fillColor}
+        tabIndex="0"
+      />
+    );
+  }
+}
```

Comment:
Since all the other shapes used are regular polygons, I was thinking this would be an equilateral triangle.  Given the length of one side (a), the height (h) of an equilateral triangle is `h = (1/2) * √3 * a`.  I'm not sure off the top of my head what this means for the space separated list of points that define the vertices of your triangle here, but let me know if you want to puzzle through it together.

Reply:
Not sure I grasp the math here when plotting this in svg space.

Reply:
Given `width` the `height = (1/2) * √3 * width` so `<polygon points="width/2,0 width,height 0,height" />`
should give you an equilateral triangle whose sides are all same length as side of the square, or radius of the circle.   I think...

Reply:
The equation worked as expected! Thank you for that!